sdk/eventhub/Azure.Messaging.EventHubs/src/Consumer/LastEnqueuedEventProperties.cs

```diff
@@ -68,5 +70,89 @@ public struct LastEnqueuedEventProperties
                  sourceEvent?.LastPartitionPropertiesRetrievalTime)
         {
         }
+
+        /// <summary>
+        ///   Determines whether the specified <see cref="EventPosition" /> is equal to this instance.
+        /// </summary>
+        ///
+        /// <param name="other">The <see cref="EventPosition" /> to compare with this instance.</param>
+        ///
+        /// <returns><c>true</c> if the specified <see cref="EventPosition" /> is equal to this instance; otherwise, <c>false</c>.</returns>
+        ///
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public bool Equals(LastEnqueuedEventProperties other)
```

Comment:
If you specifically override implement IEquatable, I would not hide the Equals method. We hide it on many types because comparing them is not a scenario.

Reply:
Roger that;  I'll tune our equatables.